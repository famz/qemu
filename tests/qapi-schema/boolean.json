{ 'true-value': true, 'false-value': false }
