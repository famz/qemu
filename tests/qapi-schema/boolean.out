[OrderedDict([('true-value', True), ('false-value', False)])]
[]
[]
